docs/rabbitmq-server.service.example

```diff
@@ -0,0 +1,17 @@
+# systemd unit example
+[Unit]
+Description=RabbitMQ broker
+After=syslog.target network.target
+
+[Service]
+Type=notify
+User=rabbitmq
+Group=rabbitmq
+NotifyAccess=all
+TimeoutStartSec=3600
```

Comment:
I discussed this with @michaelklishin and he agrees with you, so we'll keep this timeout of one hour.